lib/cloudflair/connection.rb

```diff
@@ -7,22 +9,22 @@ module Cloudflair
   class Connection
     def self.new
       config = Cloudflair.config
-
       new_faraday_from config
     end
 
     def self.headers
-      headers = {}
-      cloudflare_auth_config = Cloudflair.config.cloudflare.auth
-      if !(cloudflare_auth_config.key.nil? || cloudflare_auth_config.email.nil?)
-        headers['X-Auth-Key'] = cloudflare_auth_config.key
-        headers['X-Auth-Email'] = cloudflare_auth_config.email
-      elsif !cloudflare_auth_config.user_service_key.nil?
-        headers['X-Auth-User-Service-Key'] = cloudflare_auth_config.user_service_key
-      else
-        raise CloudflairError, 'Neither email & key nor user_service_key have been defined.'
+      {}.tap do |request_headers|
```

Comment:
I'd probably prefer something like this:
```ruby
cloudflare_auth_config = Cloudflair.config.cloudflare.auth

unless cloudflare_auth_config.key.nil? || cloudflare_auth_config.email.nil?
  return { 'X-Auth-Key' => cloudflare_auth_config.key, 'X-Auth-Email' => cloudflare_auth_config.email }
end

unless cloudflare_auth_config.user_service_key.nil?
  return { 'Authorization' => "Bearer #{cloudflare_auth_config.user_service_key}" }
end

raise CloudflairError, 'Neither email & key nor user_service_key have been defined.'
```

Or `if !` if you're not a fan of `unless`.

Reply:
That's a fine suggestion. I think I wanted to keep the structure of what was there, but I would be happier to make it look like this.

Reply:
The `user_service_key` should probably also be reflected in the `README.md`?

Reply:
Done.

Reply:
> The `user_service_key` should probably also be reflected in the `README.md`?

The Configuration section mentions it? Do we need something else?

Reply:
Nope looks perfect!